www/pages/blog.tsx

```diff
@@ -50,17 +49,32 @@ function Blog(props: any) {
     )
   }, [category])
 
+  const meta_title = 'Supabase Blog: Open Source Firebase alternative Blog'
+  const meta_description = 'Get all your Supabase News on the Supabase blog.'
+
   return (
     <>
-      <Head>
-        <link
-          rel="alternate"
-          type="application/rss+xml"
-          title="RSS feed for blog posts"
-          href={`${basePath}/rss.xml`}
-        />
-      </Head>
-      <NextSeo title="Blog" description="Latest news from the Supabase team." />
+      <NextSeo
```

Comment:
@Isaiah-Hamilton hey 👋  

I updated this page so that we're no longer using the other Head component.
What was happening was, the Next/Head component was overriding the settings being used by NextSeo component.

I think what you'll need to do is, remove the other NextHead components in the other pages.